src/libtomahawk/audio/AudioOutput.cpp

```diff
@@ -37,7 +37,7 @@
 #include <vlc/libvlc_events.h>
 #include <vlc/libvlc_version.h>
 
-static const int ABOUT_TO_FINISH_TIME = 2000;
+static const float ABOUT_TO_FINISH_POS = 0.95f;
```

Comment:
This seems dangerous. Imagine listening to an hour long track - the "about to finish" would set in 3 minutes before the end of the song.